cmd/fortifyExecuteScan_test.go

```diff
@@ -610,3 +610,37 @@ func TestAutoresolveClasspath(t *testing.T) {
 		assert.Equal(t, "some.jar;someother.jar", result, "Expected different result")
 	})
 }
+
+func TestPopulateMavenTranslate(t *testing.T) {
+	t.Run("src without translate", func(t *testing.T) {
+		config := fortifyExecuteScanOptions{Src: "./**/*"}
+		translate, err := populateMavenTranslate(&config, "")
+		assert.NoError(t, err)
+		assert.Equal(t, `[{"classpath":"","src":"./**/*"}]`, translate, "Expected different parameters")
+	})
+
+	t.Run("exclude without translate", func(t *testing.T) {
+		config := fortifyExecuteScanOptions{Exclude: "./**/*"}
+		translate, err := populateMavenTranslate(&config, "")
+		assert.NoError(t, err)
+		assert.Equal(t, `[{"classpath":"","exclude":"./**/*"}]`, translate, "Expected different parameters")
+	})
+
+}
+
+func TestPopulatePipTranslate(t *testing.T) {
+	t.Run("PythonAdditionalPath without translate", func(t *testing.T) {
+		config := fortifyExecuteScanOptions{PythonAdditionalPath: "./lib;."}
+		translate, err := populatePipTranslate(&config, "")
+		assert.NoError(t, err)
+		assert.Equal(t, `[{"pythonExcludes":"","pythonIncludes":"","pythonPath":";./lib;."}]`, translate, "Expected different parameters")
+	})
+
+	t.Run("PythonIncludes without translate", func(t *testing.T) {
+		config := fortifyExecuteScanOptions{PythonIncludes: "./**/*"}
+		translate, err := populatePipTranslate(&config, "")
+		assert.NoError(t, err)
+		assert.Equal(t, `[{"pythonExcludes":"","pythonIncludes":"./**/*","pythonPath":";"}]`, translate, "Expected different parameters")
+	})
+
```

Comment:
Here, too, perhaps one more test-case for using `config.Translate` verbatim if provided.